core/src/main/java/com/sequenceiq/cloudbreak/service/stack/flow/UpdateNodeCountValidator.java

```diff
@@ -155,7 +155,7 @@ public void validateStackStatusForStartHostGroup(StackDto stack, InstanceGroupAd
             if (!unhealthyHostGroupNames.isEmpty()) {
                 throw new BadRequestException(format("Upscaling is Not Allowed for HostGroup: '%s' as Data hub '%s' has " +
```

Comment:
This message needs to be downscale / upscale dependent.

Reply:
Yes, I will do the change.